sdk/src/test/scala/com/horizen/account/state/ForgerStakeMsgProcessorTest.scala

```diff
@@ -408,10 +408,12 @@ class ForgerStakeMsgProcessorTest
     }
   }
 
+  // this test is not meaningful anymore: all sanity checks are performed before calling any MessageProcessor
+  // ===> here we are succesful even with an empty balance account
   @Test
+  @Ignore
```

Comment:
do we need to keep this ignored test at all?

Reply:
No you are right, I removed it